src/scan.js

```diff
@@ -1,8 +1,20 @@
 const path = require('path');
 const fs = require('fs');
 
-module.exports = function (pathName, dts) {
+// ./Button-Next => ./ButtonNext
+const componentNameFormat = (str) => {
+  return str.split('-').map(subStr => upperCaseFirstChar(subStr)).join('')
+}
+
+// ./button-next => ./Button-next
+const upperCaseFirstChar = (str) => {
+  return str.charAt(0).toUpperCase() + str.slice(1)
+}
+
+module.exports = function (pathName, dts, options) {
```

Comment:
Correct me if I am wrong, but `options` was not used at all until now, right?
If that is the case, I just want to understand if we can remove it and be opinionated about the behavior with componentNameFormat or do we want to keep it configurable

Reply:
Allow such configuration sounds ok, but the API feels weird.
boolean `componentNameFormat` is not really intuitive. it should either get enummed strings or be renamed to `camelCase: true`

Moreover, please add some documentation to the readme

Reply:
@lbelinsk 
I know its kinda weird but this isn't really correct to use `camelCase: true` since we both camelcase AND uppercase first char since this is a component export (at least that is the way we export today from ui-core). 

Our first approach was two options (`camelcase: true` and `uppercaeseFirstChar: true`) but that felt clunky...

what do you think?

Reply:
I agree with @JoMarton . In the meanwhile I added to the docs and explanation with an example for the options format pattern. If we decide about something different ill change it.